CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ include(cmake/xcode.cmake)
 CMAKE_DEPENDENT_OPTION(KTX_FEATURE_TOOLS "Create KTX tools" ON "NOT IOS;NOT ANDROID;NOT EMSCRIPTEN" OFF)
 option( KTX_FEATURE_DOC "Create KTX documentation" OFF )
 option( KTX_FEATURE_LOADTEST_APPS "Create load tests apps that load and display various KTX textures" OFF )
-option( KTX_FEATURE_FORCE_STATIC "Always create static libraries" OFF )
+CMAKE_DEPENDENT_OPTION(KTX_FEATURE_STATIC_LIBRARY "Create static libraries (shared otherwise)" OFF "NOT IOS;NOT EMSCRIPTEN" ON)
```

Comment:
`NOT IOS; NOT EMPSCRIPTEN`? It looks like this should be setting `KTX_FEATURE_STATIC_LIBRARY` ON for IOS and EMSCRIPTEN so why the NOT?

Reply:
Yeah, the makro is a bit confusing to decipher. In words this means:
"if `NOT IOS` AND `NOT EMSCRIPTEN` enable the option with the default `OFF`. Set it to `ON` otherwise."

Reply:
See `KTX_FEATURE_TOOLS` above for comparison

Reply:
Thanks for the explanation.